v20.1/query-behavior-troubleshooting.md

```diff
@@ -63,6 +63,32 @@ High latency SQL statements are displayed on the [**Statements page**](admin-ui-
 
 You can also check the [service latency graph](admin-ui-sql-dashboard.html#service-latency-sql-99th-percentile) and the [CPU graph](admin-ui-hardware-dashboard.html#cpu-percent) on the SQL and Hardware Dashboards, respectively. If the graphs show latency spikes or CPU usage spikes, these might indicate slow queries in your cluster.
 
+## Visualize statement traces in Jaeger
+
+You can look more closely at the behavior of a statement by visualizing a statement trace in [Jaeger](https://www.jaegertracing.io/). A [statement trace](show-trace.html) contains messages and timing information from all nodes involved in the execution.
+
+1. Activate [statement diagnostics](admin-ui-statements-page.html#diagnostics) on the Admin UI Statements Page or run [`EXPLAIN ANALYZE (DEBUG)`](explain-analyze.html#debug-option) to obtain a diagnostics bundle for the statement.
+
+1. [Access the Jaeger UI.](https://www.jaegertracing.io/docs/1.20/getting-started/)
```

Comment:
I think it'd be useful to paste here the command for starting Jaeger using Docker, and showing an example url (http://localhost:16686/search)